vector/src/main/java/im/vector/app/core/preference/VectorCheckboxPreference.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.core.preference
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.TextView
+import androidx.preference.CheckBoxPreference
+import androidx.preference.PreferenceViewHolder
+
+open class VectorCheckboxPreference : CheckBoxPreference {
```

Comment:
This class does not have to be `open`. It does not have to be sealed as well, but let's keep the default Kotlin behavior (so seal) unless we have to extend the class.